src/pages/tutorials/aiops/installation/index.mdx

```diff
@@ -70,36 +69,62 @@ You can check your entitlement for Watson AIOPs by clicking **View Library**
 
 ![](images/entitlement_check.png)
 
-**NOTE:** If you do not have an existing IBM contract that allows you access to the software for these labs, you can register for trial using link [Trial license of Watson AIOps](https://www.ibm.com/account/reg/us-en/login?formid=urx-51074)
+**NOTE:** If your entitlement key is not valid for Watson for AIOps Event Manager, you can register for a 60 day trial using the following link: [Request Watson AIOps trial](https://www.ibm.com/account/reg/us-en/login?formid=urx-51074).
 
 ## 2-4: Prepare your OpenShift cluster
 
-This step involves creating a namespace to install the Watson for AIOps Event Manager Operator into, and setting up your software entitlement key, so that you will be able to deploy an instance of Event Manager.
+This step involves creating a namespace (also known as a "Project" on OpenShift) to install the Watson for AIOps Event Manager Operator into, and setting up your software entitlement key, so that you will be able to deploy an instance of Event Manager.
 
-This step involves the following tasks:
+This section involves the following tasks:
 
-- Open the **OpenShift UI**
-- Acquire a token and log in to the OpenShift cluster via the command line
+- Navigate to and open the **OpenShift UI**
+- Establish a connection to your control virtual machine using the supplied SSH key
+- Acquire an OpenShift login token and log in to the OpenShift cluster via the command line
 - Create a **namespace** to deploy Event Manager into
 - Create an **entitlement key secret**
 
-Go to your newly created OpenShift cluster and click OpenShift web console to open the main OpenShift UI:
+During this module, you will need to log in to your dedicated "control" virtual machine, to run configuration commands against your OpenShift cluster.
```

Comment:
Would suggest removing this section. It's very specific to IBM led tech jam events, but a user coming to this repo offline or outside an event will not get these emails. 

I'm working @lindatan878 to cover the emails at the beginning of tech jam events. It will include an arch diagram showing the manage-to / manage-from clusters.